sorc/chgres_cube.fd/input_data.F90

```diff
@@ -6429,6 +6422,11 @@ subroutine read_grib_soil(the_file,inv_file,vname,vname_file,dummy3d,rc)
 
  end subroutine read_grib_soil
 
+!---------------------------------------------------------------------------
```

Comment:
All these lines with "--------------------------" should be removed everywhere.

Reply:
I use "--------------------" all through my codes to block comments.  I removed them from any doxygen comments.  But not from the general Fortran comments.